### ISSUE TITLE
SmartMenu keyboard fails with JAWS

### ISSUE THREAD

**Issue by blobula** [vadikom/smartmenus#77]
When Jaws is enabled the keyboard interaction reverts back to just tab/enter/space, no arrow key support. Focus of items is not displayed. I see the same behavior on the official demo page as well. Is there a fix for this, or any idea why this is happening?

**Comment by vadikom** [comment]
In JAWS (and other screen readers) the default method for browsing a list of links is honored and not overridden so that users are able to use their familiar key strokes to navigate the menu tree.

If you like, in JAWS particularly (not sure about all other screen readers) you could wrap the menu tree in a container element with a `role="application"` attribute to override JAWS' commands and make the keyboard add-on behave as normally but this is not recommended really since all JAWS users are familiar with its default key shortcuts for browsing nested list of links and this would only confuse them.

As a side note, if you'd like to provide better usability for JAWS users, you could wrap the root level links in heading <h*> tags since it has dedicated shortcuts for quickly jumping through headings so whenever a root item is expanded, users could still quickly jump to the next root item. But even if you don't do that, the sub menus are also configured to be hidden to JAWS by default so users could quickly tab through the root links before activating any of them - so by default, the script is already designed to be accessible and usable to screen reader users.
